.tool-versions

```diff
@@ -0,0 +1 @@
+golang 1.17.11
```

Comment:
Can you add the `.tool-versions` to `.gitignore` and drop this file from this PR?